transport_plugins/jlink/RELEASE.md

```diff
@@ -3,6 +3,15 @@
 All major changes in each released version of the jlink transport plugin are
 listed here.
 
+## 1.0.6
```

Comment:
Same comment about versions here -- make sure you're aligned on the version.py and this. Multiple things can go in to one release.

Reply:
Fixed.